app/models/user.js

```diff
@@ -30,15 +35,21 @@ export default DS.Model.extend({
     let warningSubs = [];
     let subs = this.get('subordinates');
     subs.forEach(sub => {
-      warningSubs.addObjects(sub.get('warningSubordinates'));      
+      warningSubs.addObjects(sub.get('warningSubordinates'));
       if (sub.get('showInWarning')) {
         warningSubs.addObject(sub);
       }
     });
     return warningSubs;
   }),
-  
+
   fullName: Ember.computed('firstName', 'lastName', function() {
     return `${this.get('lastName')}, ${this.get('firstName')}`;
   }),
+
+  intervene: Ember.computed('rateNo', function() {
```

Comment:
@meilinantazo i think we won't need this, in the tm view the "I'll Intervene" link is always present so we can just hardcode it.. will adjust this in the next PR shortly :)